package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"dependencies": {
-		"globby": "^13.1.4",
```

Comment:
Hmmm, From [Change ](https://github.com/sindresorhus/globby/tree/v14.0.0) look fine to me there is no breaking change here.